roles/create_vms/tasks/provision_vms.yml

```diff
@@ -25,5 +25,7 @@
       loop: "{{ kvm_nodes }}"
 
     - name: Run vm creation_scripts
-      shell: "/bin/bash {{ vm_create_scripts_dir }}/{{ item.name }}_setup_vm.sh"
+      shell:  # noqa: no-changed-when
+        cmd: "/bin/bash {{ vm_create_scripts_dir }}/{{ item.name }}_setup_vm.sh"
```

Comment:
I think this will always produce a _changed_ due to the fact it needs to run the shell command.